CMakeLists.txt

```diff
@@ -1,25 +1,19 @@
-cmake_minimum_required(VERSION 3.10.2)
-# remove cmake/modules/FindPython* once 3.12 is required
```

Comment:
@tchaikov do we plan on removing these as well, if that is still remaining work, can we keep a note of it somewhere